wpilibc/src/main/native/cpp/DigitalInput.cpp

```diff
@@ -73,5 +73,6 @@ int DigitalInput::GetChannel() const { return m_channel; }
 
 void DigitalInput::InitSendable(SendableBuilder& builder) {
   builder.SetSmartDashboardType("Digital Input");
-  builder.AddBooleanProperty("Value", [=]() { return Get(); }, nullptr);
+  builder.AddBooleanProperty(
+      "Value", [=]() { return Get(); }, nullptr);
```

Comment:
Was there an option that changed that reformatted these to always break after the first paren?  While it makes sense for multi-line, some of these fit on one line just fine.

Reply:
I just pushed an updated .clang-format based on clang-format 9.0's Google config dump. The new option `AllowAllArgumentsOnNextLine: true` seems relevant, but setting that to `false` doesn't change the formatting at all.

Reply:
Seems to only do it if there's a lambda with an argument on either side, as with
```
  builder.AddBooleanProperty(
      1, [] {}, 2);
```
Removing the first or third arguments or the capture (making the second arg a braced list) has the desired formatting.

Reply:
This is the relevant change: https://reviews.llvm.org/D40988